toml/toml.go

```diff
@@ -36,7 +36,7 @@ func (d *Duration) UnmarshalText(text []byte) error {
 
 // MarshalText converts a duration to a string for decoding toml
 func (d Duration) MarshalText() (text []byte, err error) {
-	return []byte(d.String()), nil
+	return []byte(fmt.Sprintf(`"%s"`, d.String())), nil
```

Comment:
Duration strings were written without quotes in the toml output like this `  election-timeout = 1s`. The toml parser was choking on it so I changed the Marshal function to quote them.